website/docs/cloud-docs/no-code-provisioning/module-design.mdx

```diff
@@ -46,6 +47,15 @@ For example, you could build modules that satisfy the following well-scoped use
 - Deploying all resources needed for a three-tier web application
 - Deploying a database with constraints on resource allocation and deployment region
 
+### Adjust for New Module Versions
+No-code provisioning desgination is now pinned to a selected version of a module. You must select the version at the time of enabling the module. If no selection is made, it will default to the latest version at the time of enabling. This ensures that you can define variable input options for the intended version and that no-code users provision the most appropriate version. When a newer version of the module becomes avialable, you may want to update the pinned no-code version to the latest.
```

Comment:
```suggestion
No-code provisioning designation is pinned to a selected version of a module. You must select the version at the time of enabling the module. If no selection is made, it will default to the latest version at the time of enablement. This ensures that you can define variable input options for the intended version and that no-code users provision the most appropriate version. When a newer version of the module becomes available, you may decide to update the pinned no-code version to the latest.
```

Reply:
@opslivia Did you see my suggestion here? If this doesn't feel good I can take another try at rewriting what is currently there! (I'm unresolving this comment for now)

Reply:
(Closing because I resuggested these in a new review)